### DISCUSSION TITLE
Show time of last comment in discussion list

### DISCUSSION THREAD

**Discussion by michicc** [community/community#3099] [Discussions] [answered]
The list of discussions currently shows for each discussion who started it when, and how many comments it has.

A IMHO important info is the time of the last comment, which is apparently not shown so far. It would help to quickly see at a glance if you've missed anything, especially if somebody doesn't want to subscribe to each and every discussion.

**Comment by evi-liu** [comment, marked as answer]
Thanks for the feedback! We are tracking this request on our end.
